doc/source/whatsnew/v0.17.0.txt

```diff
@@ -21,6 +21,15 @@ Check the :ref:`API Changes <whatsnew_0170.api>` and :ref:`deprecations <whatsne
 New features
 ~~~~~~~~~~~~
 
+Plot submethods
+^^^^^^^^^^^^^^^
+
+The Series and DataFrame ``.plot()`` method allows for customizing :ref:`plot types<visualization.other>` by supplying the ``kind`` keyword arguments. Unfortunately, many of these kinds of plots use different required and optional keyword arguments, which is extremely difficult to document what any given plot kind uses out of the dozens of possible keyword arguments.
+
+To alleviate this issue, we have added a new, optional plotting interface, which exposes each kind of plot as a method of the ``.plot`` attribute. Instead of writing ``series.plot(kind=<kind>, ...)``, you can now also use ``series.plot.<kind>(...)``.
+
+A major advantage of this approach is that required arguments for each plotting method now can appear first in the function signature, and can be referred to by position, e.g., you can write ``series.plot.pie('food')`` instead of ``series.plot(y='food', kind='pie')``.
+
```

Comment:
couple of examples in a code-block maybe? (and maybe an ipython tab completion example)